.github/workflows/pull-request.yaml

```diff
@@ -18,14 +18,15 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Check for ticket number
-        uses: gsactions/commit-message-checker@v2
-        with:
-          pattern: '^N8N-\d+\s'
-          error: 'Your first line has to start with a ticket number of the issue like "N8N-1".'
-          excludeDescription: 'true'
-          checkAllCommitMessages: 'true'
-          accessToken: ${{ secrets.GITHUB_TOKEN }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+
+      - name: Install dependencies
+        run: npm install --save-dev @commitlint/config-nx-scopes
```

Comment:
Install globally

Reply:
Done.

I needed to replace the used GitHub action with a more "manual" approach based on the commintlint [documentation](https://commitlint.js.org/guides/ci-setup.html#github-actions) for GitHub actions, otherwise, the correct usage of additional commitlint configuration packages where not possible.

Other actions that tell they would do the job are mostly locked to ancient versions of commitlint without the ability to overwrite the requested version and were also locked to a small subset of commitlint config packages.